readme.md

```diff
@@ -117,6 +117,18 @@ Capturing a context snapshot and reactivating it in another thread is no differe
 For insight, the library tracks the overall time used creating and reactivating
 context snapshots along with time spent in each individual `ContextManager`.
 
+## Building uber-jars / jars with dependencies
```

Comment:
Can you move this section just above `Performance metrics`, so that isn't separated from logging performance and metrics reporting?

Reply:
Ah, sorry yes that paragraph is misplaced.